WebToPay.php

```diff
@@ -2320,6 +2346,33 @@ public function encodeSafeUrlBase64($text) {
         return strtr(base64_encode($text), array('+' => '-', '/' => '_'));
     }
 
+    /**
+     * Decrypts string with aes-256-gcm algorithm
+     *
+     * @param $stringToDecrypt string
+     * @param $key string
+     *
+     * @return string|false
+     */
+    function decryptGCM($stringToDecrypt, $key) {
```

Comment:
Looks like you are not using this function anywhere?

Reply:
WebToPay.php:1270 - it is used there